lib/install/is-dev.js

```diff
@@ -2,6 +2,26 @@
 var isDev = exports.isDev = function (node) {
   return node.package._requiredBy.some(function (req) { return req === '#DEV:/' })
 }
-exports.isOnlyDev = function (node) {
+var isOnlyDev = exports.isOnlyDev = function (node) {
```

Comment:
@iarna sorry for the double comment :crying_cat_face: The other export, `isDev` is only used by is-extraneous.js, but after a quick glance, it doesn't look necessary. Mind taking a look as well?